python/src/puml/generate_plantuml_diagrams_impl.py

```diff
@@ -191,33 +201,34 @@ def puml_sequence(architecture_file: str, output_directory: str, classification:
             "sequences": sequences,
             "classification": classification
         }
+        yaml_list.append([properties])
```

Comment:
Something about this isn't going through correctly for sequence, it is only getting the last sorted model, I am currently poking at it.

Reply:
Got it working. If you'll look over the changes from the last two commits I put up, please. @bunchw